dist/preview release/what's new.md

```diff
@@ -134,6 +134,7 @@
 - Added LoadScriptAsync tools helper function [MackeyK24](https://github.com/mackeyk24))  
 - Added customShaderNameResolve to PBRMaterialBase to allow subclasses to specify custom shader information [MackeyK24](https://github.com/mackeyk24))
 - Added PBRCustomMaterial to material library to allow easy subclassing of PBR materials [MackeyK24](https://github.com/mackeyk24))
+- Added LoadSkinAsync to GLTF2 loaders to allow custom skinning support [MackeyK24](https://github.com/mackeyk24))
```

Comment:
Should probably remove this until the underscore is removed (i.e. the contract is solid)
```suggestion
```